saleor/graphql/product/tests/test_product_channel_listings_queries.py

```diff
@@ -139,6 +138,306 @@ def test_product_channel_listing_pricing_field_no_address(
+
+    # then
+    data = None
+    for channel_data in content["data"]["product"]["channelListings"]:
+        if channel_data["channel"]["slug"] == channel_PLN.slug:
+            data = channel_data
+
+    price_range_PL = data["pricingPL"]["priceRange"]
+    price_range_undiscounted_PL = data["pricingPL"]["priceRangeUndiscounted"]
+    assert price_range_PL["start"]["net"]["amount"] == net_PL
+    assert price_range_PL["start"]["gross"]["amount"] == gross_PL
+    assert price_range_PL["stop"]["net"]["amount"] == net_PL
+    assert price_range_PL["stop"]["gross"]["amount"] == gross_PL
+    assert price_range_undiscounted_PL["start"]["net"]["amount"] == net_PL
+    assert price_range_undiscounted_PL["start"]["gross"]["amount"] == gross_PL
+    assert price_range_undiscounted_PL["stop"]["net"]["amount"] == net_PL
+    assert price_range_undiscounted_PL["stop"]["gross"]["amount"] == gross_PL
+
+    price_range_DE = data["pricingDE"]["priceRange"]
+    price_range_undiscounted_DE = data["pricingDE"]["priceRangeUndiscounted"]
+    assert price_range_DE["start"]["net"]["amount"] == net_DE
+    assert price_range_DE["start"]["gross"]["amount"] == gross_DE
+    assert price_range_DE["stop"]["net"]["amount"] == net_DE
+    assert price_range_DE["stop"]["gross"]["amount"] == gross_DE
+    assert price_range_undiscounted_DE["start"]["net"]["amount"] == net_DE
+    assert price_range_undiscounted_DE["start"]["gross"]["amount"] == gross_DE
+    assert price_range_undiscounted_DE["stop"]["net"]["amount"] == net_DE
+    assert price_range_undiscounted_DE["stop"]["gross"]["amount"] == gross_DE
+
+
+def test_product_pricing_default_country_default_rate(
+    product_available_in_many_channels,
+    channel_PLN,
+    staff_api_client,
+    permission_manage_products,
+):
+    # given
+    product = product_available_in_many_channels
+    _enable_flat_rates(channel_PLN, True)
+    TaxClassCountryRate.objects.all().delete()
+    TaxClassCountryRate.objects.create(country=channel_PLN.default_country, rate=23)
+
+    # when
+    variables = {
+        "id": graphene.Node.to_global_id("Product", product.id),
+        "channel": channel_PLN.slug,
+    }
+    response = staff_api_client.post_graphql(
+        QUERY_PRODUCT_PRICING,
+        variables,
+        permissions=(permission_manage_products,),
+        check_no_permissions=False,
+    )
+    content = get_graphql_content(response)
+
+    # then
+    data = None
+    for channel_data in content["data"]["product"]["channelListings"]:
+        if channel_data["channel"]["slug"] == channel_PLN.slug:
+            data = channel_data
+
+    price_range = data["pricing"]["priceRange"]
+    price_range_undiscounted = data["pricing"]["priceRangeUndiscounted"]
+    assert price_range["start"]["net"]["amount"] == 40.65
+    assert price_range["start"]["gross"]["amount"] == 50.00
+    assert price_range["stop"]["net"]["amount"] == 40.65
+    assert price_range["stop"]["gross"]["amount"] == 50.00
```

Comment:
IMO we should calculate those values on the right, or get them from some object fields. Right now these values are a kind of magic and it will be hard to keep those tests.